ci/pipeline-vps.yml

```diff
@@ -0,0 +1,81 @@
+resources:
```

Comment:
If this is the docs PR, why does it include pipeline stuff?

Perhaps you need to resubmit with correct commit?